libraries/ESP8266WebServer/examples/FSBrowser/FSBrowser.ino

```diff
@@ -257,54 +628,48 @@ void setup(void) {
   DBG_OUTPUT_PORT.print("Connected! IP address: ");
   DBG_OUTPUT_PORT.println(WiFi.localIP());
 
-  MDNS.begin(host);
-  DBG_OUTPUT_PORT.print("Open http://");
-  DBG_OUTPUT_PORT.print(host);
-  DBG_OUTPUT_PORT.println(".local/edit to see the file browser");
+  ////////////////////////////////
+  // MDNS INIT
+  if (MDNS.begin(host)) {
+    MDNS.addService("http", "tcp", 80);
+    DBG_OUTPUT_PORT.print("Open http://");
+    DBG_OUTPUT_PORT.print(host);
+    DBG_OUTPUT_PORT.println(".local/edit to open the FileSystem Browser");
+  }
+
+  ////////////////////////////////
+  // WEB SERVER INIT
 
+  // Filesystem status
+  server.on("/status", HTTP_GET, handleStatus);
 
-  //SERVER INIT
-  //list directory
+  // List directory
   server.on("/list", HTTP_GET, handleFileList);
-  //load editor
-  server.on("/edit", HTTP_GET, []() {
```

Comment:
I seem to remember there was a reason for this special handling, but I can't for the life of me remember what it was.

Reply:
Err, that special handler is still there, I just moved it up so that the GET method appears before PUT, DELETE and POST. Sounded more logical to me...

Reply:
I meant the lambda wrapper. NVM, if something breaks we can handle it in a later PR.

Reply:
Oh I get it now. Well I thought the lambda was used because the code was so simple it didn't justify the overhead of a named function.
As the code became a bit more complex with the option to return the embedded binary version, I found it cleaner to create a dedicated handler function like the other calls.